apps/sms/js/thread_list_ui.js

```diff
@@ -646,7 +648,7 @@ var ThreadListUI = {
     Settings.setReadAheadThreadRetrieval(this.FIRST_PANEL_THREAD_COUNT);
   },
 
-  renderThreads: function thlui_renderThreads(firstViewDoneCb, allDoneCb) {
+  renderThreads: function thlui_renderThreads(firstViewDoneCb) {
```

Comment:
maybe https://groups.google.com/forum/#!topic/mozilla.dev.webapi/N1FjiEqCflI ? It seems like they don't want ProgressPromises after all... they want streams. But here just having 2 promises is good enough. (but maybe not for this patch)